src/CLIMA.jl

```diff
@@ -1,5 +1,9 @@
 module CLIMA
 
+const CLIMA_MAJOR_VERSION = 0
+const CLIMA_MINOR_VERSION = 0
+const CLIMA_VERSION = VersionNumber(CLIMA_MAJOR_VERSION, CLIMA_MINOR_VERSION)
```

Comment:
Can we not get this from `Pkg` somehow?

Reply:
I don't know. Can we @simonbyrne?

Reply:
We can always update this later, so long this information is defined in a single place. Also, looks like we'll have an [experimental way](https://julialang.github.io/Pkg.jl/v1/api/#Pkg.project) to get this in 1.4.

Reply:
I'm not sure that would even work, since CLIMA could be a dependency of a different project.

Reply:
```suggestion
using Pkg.TOML
const CLIMA_VERSION = VersionNumber(TOML.parsefile(joinpath(dirname(@__DIR__),"Project.toml"))["version"])
```